src/intTest/resources/inbound_uat_data/acceptance/1.fhir.json

```diff
@@ -1,6 +1,10 @@
 {
   "resourceType": "Parameters",
   "parameter": [
+    {
+      "name": "gpTradingPartnerCode",
+      "valueString": "XX11"
```

Comment:
Should be TES5 if this is to match the .dat file

UNB+UNOA:2+XX11+TES5+920125:1235+00000003'

XX11 = sender
TES5 = recipient

Since this is an inbound message the recipient is the gp, so gpTradingPartnerCode should be TES5

Reply:
Actually - now that you've added 'approval' can you remove this 'acceptance' folder? This is no inbound acceptance transaction.

Reply:
This is dummy test as we don't have acceptance UAT test, but I'll change it as you say

Reply:
ah, ok I can remove that